docs/getting-started/installation/proxmox.mdx

```diff
@@ -0,0 +1,67 @@
+# Proxmox
+
+This guide covers how to install homarr inside of a linux-container in ProxmoxVE, using a fully automated approach.
+
+# What is community-scripts?
+
+https://community-scripts.github.io/ProxmoxVE/
+
+We are a community-driven initiative that simplifies the setup of Proxmox Virtual Environment (VE).
+With hundreds of scripts to help you manage your Proxmox VE environment. Whether you're a seasoned user or a newcomer, we've got you covered.
+
+# Installation
+
+It's as simple as pasting the following snippet into your Proxmox Node's console
+
+```bash
+bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/homarr.sh)"
+```
+
+:::warning
+Never trust any script directly injected from the web, without prior looking through it or searching the web for other's opinions.
+
+After executing this command, you will get asked, if you want to accept defaults or would like to specify things like IP-Adress, DNS etc, then choose advanced.
```

Comment:
Instead of glossing over those options, why not document them here and what values can be chosen or how they can be chosen (e.g. by keyboard?)